lib/services/livesync/android-device-livesync-service-base.ts

```diff
@@ -30,7 +30,7 @@ export abstract class AndroidDeviceLiveSyncServiceBase extends DeviceLiveSyncSer
 		const deviceHashService = this.getDeviceHashService(deviceAppData.appIdentifier);
```

Comment:
we have some duplicate logic in this file and in the AndroidDeviceFileSystem. Instead of calling a method here, you can just call:
```TypeScript
this.device.fileSystem.getDeviceHashService(deviceAppData.appIdentifier);
```

With the current implementation we have duplicate cashing of the Hash services